docs/concepts/credentials/3_terms-and-quote.md

```diff
@@ -3,36 +3,31 @@ id: requesting-terms-and-attestations
 title: Requesting Terms and Attestations
 ---
 
-If an attester has an established reputation for issuing attestations, the claimer may choose to submit a claim to be attested without going through the process of agreeing to terms.
-In that case, they may submit a request following this process.
-
-Otherwise, either the attester or the claimer may request the terms of the attestation, i.e., the requirements set by the participants (the claimer and the attester) that set the conditions to attest a claim.
+The claimer may request and / or the attester may send the terms of the attestation, i.e., the requirements set by the participants (the claimer and the attester) for the conditions of the attestation.
 
 These terms are defined and agreed upon before the attestation is created. This part of the process requires interaction and communication between both parties. This communication can be done independently, e.g. in person, via messaging, on social media etc., or via the KILT software development kit (SDK).
 
 ## Sending terms using the KILT SDK
 
-Both “request terms” and “submit terms” are part of the messaging: the message is sent as “request terms” and received as “submit terms”.
+Both “request terms” and “submit terms” are part of the messaging system: the message is sent as “request terms” and received as “submit terms”.
 
 The interaction is as follows:
 
-- The claimer creates an `IPartialClaims` from an attester and sends an `IRequestTerms` message
-- An attester creates an `ITerms` object and sends it as part of a submit terms message `ISubmitTerms` to the claimer.
-- The claimer receives the submit terms message, checks the terms and, if all is in order, agrees to them.
-
-Either a claimer or an attester can initiate a request for terms.
+- The claimer creates optionally a partial `Claim` and sends an message to the attester questing the terms
+- An attester creates an `Terms` object and sends it as part of a submit terms message back to the claimer.
+- The claimer receives the terms message, checks the terms and, if all is in order, agrees to them
 
 ## How to create a terms object
 
-When creating a terms object, at least one of the following must be complete: Legitimations, Quote or Delegations.
+When creating a terms object, at least one of the following must be set: Legitimations, Quote or Delegations.
 
-- **Legitimations**: Legitimations are composed of attested claims that the attester has issued previously, showing that the attester has the authority or legitimacy to attest the claim requested. This is a way of establishing trust between the participants.
+- **Legitimations**: Legitimations are Credentials, issued to the attester the claimer wants to interact with, showing that the attester has the authority or legitimacy to attest the claim requested. This is a way of establishing trust between the participants.
```

Comment:
We're using attester and claimer lowercase here. I guess we will have an editorial round afterwards once we define all the style guidelines.

Reply:
317bd4334a1cbfdb4f1aa7b1bbd7b7e8b7f2dd71